pico_limbo/src/configuration/boss_bar.rs

```diff
@@ -0,0 +1,89 @@
+use minecraft_packets::play::boss_bar_packet::{BossBarColor, BossBarDivision};
+use serde::de::Error;
+use serde::{Deserialize, Deserializer, Serialize, Serializer};
+
+#[derive(Deserialize, Serialize, Default)]
+#[serde(rename_all = "lowercase")]
+pub enum BossBarColorConfig {
+    #[default]
+    Pink,
+    Blue,
+    Red,
+    Green,
+    Yellow,
+    Purple,
+    White,
+}
+
+#[derive(Debug, Clone, Copy)]
+pub struct BossBarDivisionSerde(pub BossBarDivision);
```

Comment:
Serde? I think you could use an equivalent of the `BossBarDivision` enum directly here, similar to how the Color is handled